src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -243,10 +241,6 @@ public void initialize() {
 
     @Override
     public void dispose() {
```

Comment:
Wouldn't it be cleaner to also unregister the listeners from the networkManager here?